core/src/commonMain/kotlin/com/xebia/functional/xef/llm/Chat.kt

```diff
@@ -136,6 +99,23 @@ interface Chat : LLM {
   ): List<String> =
     promptMessages(Prompt(question), context, conversationId, functions, promptConfiguration)
 
+  @AiDsl
+  suspend fun promptMessages(
```

Comment:
I see that we are passing a `prompt` in this function. Should the function be renamed to `promptMessage`?

Reply:
I think when these methods were created the plural was related to the return type, not to the first parameter

`promptMessage` returns `String`

`promptMessages` returns `List<String>`